Source/Documentation/Manual/software-platform.rst

```diff
@@ -101,15 +102,26 @@ the software will contribute to better overall performance -- potentially
 allowing high polygon models with densely populated routes at acceptable 
 frame rates.
 
+.. _web-server:
+
 Web Server
 ==========
 
 .. _sample-web-pages:
 
+The game uses a built-in web-server to deliver standard and custom web-pages
+to any browser. This can be running on the same PC as Open Rails or another PC 
+or other device which is connected to your local network.
+
+The simplest way to access these samples is to start the game and then launch
+a browser on the smae PC. Then enter "localhost:2150" into your browser address bar. 
```

Comment:
```suggestion
a browser on the same PC. Then enter "localhost:2150" into your browser address bar. 
```
I've just found another typo. Sorry for not having spotted this previously

Reply:
I've noticed that my typing is getting less reliable as I am getting older.
Thanks for spotting this.